rfcs/0222-allow-script-multiple-matches-on-identical-code/0222-allow-script-multiple-matches-on-identical-code.md

```diff
@@ -1 +1,93 @@
-In review, see <https://github.com/nervosnetwork/rfcs/pull/222>
+---
+Number: "0222"
```

Comment:
There's never been any formal discussion on changing numbering rules, I think we should stick to the existing rule.

Reply:
Resolved, please review the PR again or simply dismiss the changes request.